README.rst

```diff
@@ -3,31 +3,33 @@ djangocms-text-ckeditor
 
 Text Plugin for django-cms with CK-Editor.
 
-It currently supports:
+The latest Version of this package supports:
 
-* Django 1.6 - 1.8
-* django CMS 3.0 - 3.1
+* Django 1.8
+* django CMS 3.2+
 
 .. WARNING::
-   ``cms.plugins.text`` and ``djangocms-text-ckeditor`` can't be used at the same time.
+   For django CMS 3.0 and 3.1 use ``djangocms-text-ckeditor`` <= 2.7 (e.g.: version 2.7.0).
 
 .. WARNING::
-   For Django 1.4 and 1.5 use ``djangocms-text-ckeditor`` < 2.7.
```

Comment:
We need to keep this. 2.7 does not suppot Django 1.4 / 1.5 whatever the CMS version